python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py

```diff
@@ -369,90 +371,38 @@ def _scatterPlot(
-
-            toPlotList.append(
-                (
-                    data["xStars"],
-                    data["yStars"],
-                    data["starsHighSNMask"],
-                    data["starsLowSNMask"],
-                    "midnightblue",
-                    newBlues,
-                    highStats,
-                    lowStats,
-                )
-            )
-        if "galaxies" in self.plotTypes:  # type: ignore
-            highArgs = {}
-            lowArgs = {}
-            for name in self._stats:
-                highArgs[name] = cast(Scalar, data[f"{{band}}_highSNGalaxies_{name}".format(**kwargs)])
-                lowArgs[name] = cast(Scalar, data[f"{{band}}_lowSNGalaxies_{name}".format(**kwargs)])
-            highStats = _StatsContainer(**highArgs)
-            lowStats = _StatsContainer(**lowArgs)
-
-            toPlotList.append(
-                (
-                    data["xGalaxies"],
-                    data["yGalaxies"],
-                    data["galaxiesHighSNMask"],
-                    data["galaxiesLowSNMask"],
-                    "firebrick",
-                    newReds,
-                    highStats,
-                    lowStats,
-                )
-            )
-        if "unknown" in self.plotTypes:  # type: ignore
-            highArgs = {}
-            lowArgs = {}
-            for name in self._stats:
-                highArgs[name] = cast(Scalar, data[f"{{band}}_highSNUnknown_{name}".format(**kwargs)])
-                lowArgs[name] = cast(Scalar, data[f"{{band}}_lowSNUnknown_{name}".format(**kwargs)])
-            highStats = _StatsContainer(**highArgs)
-            lowStats = _StatsContainer(**lowArgs)
-
-            toPlotList.append(
-                (
-                    data["xUnknown"],
-                    data["yUnknown"],
-                    data["unknownHighSNMask"],
-                    data["unknownLowSNMask"],
-                    "green",
-                    None,
-                    highStats,
-                    lowStats,
-                )
-            )
-        if "any" in self.plotTypes:  # type: ignore
-            highArgs = {}
-            lowArgs = {}
-            for name in self._stats:
-                highArgs[name] = cast(Scalar, data[f"{{band}}_highSNUnknown_{name}".format(**kwargs)])
-                lowArgs[name] = cast(Scalar, data[f"{{band}}_lowSNUnknown_{name}".format(**kwargs)])
-            highStats = _StatsContainer(**highArgs)
-            lowStats = _StatsContainer(**lowArgs)
-
-            toPlotList.append(
-                (
-                    data["x"],
-                    data["y"],
-                    data["anyHighSNMask"],
-                    data["anyLowSNMask"],
-                    "purple",
-                    None,
-                    highStats,
-                    lowStats,
+
+        colormaps_datatype = {
+            "stars": newBlues,
+            "galaxies": newReds,
+        }
```

Comment:
There should be colour maps for the other types, even if they are just set to None for now and fixed later.

Reply:
I added the colormap as an attribute but couldn't find colormaps for unknown or all/any. There is a divergent one with three colors in skyPlot.py.